### DISCUSSION TITLE
Dealing with ecchanges file

### DISCUSSION THREAD

**Discussion by nn15296** [iTwin/itwinjs-core#1175] [Q&A] [answered]
I'm using the BriefcaseManager.processChangeSets function to update a briefcase to later changesets and then extracting the change summaries. In the picture attached, the briefcase size is 3GB. The ECCHANGES file after having processed many of the changesets is ~14GB. Is there any official way to deal with this file growth or should I just delete the file using the"fs" module after updating and processing each changeset to prevent the accumulation?

![image](https://user-images.githubusercontent.com/33571096/114391500-7a55bd00-9b8f-11eb-9732-e6ffcf9bf046.png)

Thank you

**Comment by ramanujam-raman-zz** [comment, marked as answer]
Please delete it using the node API for now. We will add a better way to do this.
